src/mpd.c

```diff
@@ -3657,6 +3668,7 @@ mpd_command_enableoutput(struct evbuffer *evbuf, int argc, char **argv, char **e
     }
 
   memset(&param, 0, sizeof(struct output_get_param));
+  param.curid = 0;
```

Comment:
Not removed? Same below

Reply:
there were 3, I thought I got them all, bleh